Doc/tutorial/inputoutput.rst

```diff
@@ -300,11 +300,14 @@ writing. The *mode* argument is optional; ``'r'`` will be assumed if it's
 omitted.
 
 Normally, files are opened in :dfn:`text mode`, that means, you read and write
-strings from and to the file, which are encoded in a specific encoding. If
-encoding is not specified, the default is platform dependent (see
-:func:`open`). ``'b'`` appended to the mode opens the file in
-:dfn:`binary mode`: now the data is read and written in the form of bytes
-objects.  This mode should be used for all files that don't contain text.
+strings from and to the file, which are encoded in a specific *encoding*.
+If *encoding* is not specified, the default is platform dependent
+(see :func:`open`).
+But passing ``encoding="utf-8"`` is highly recommended because
+UTF-8 is the most commonly used encoding for now.
+``'b'`` appended to the mode opens the file in :dfn:`binary mode`:
```

Comment:
```
Appending a ``'b'`` to the mode opens the file in :dfn:`binary mode`. Binary mode data is read and written as ``bytes`` objects without use of a codec.
```

Reply:
I'd use "encoding" instead of "codec". I don't see "codec" used anywhere else in this file.

Reply:
(I didn't rewrite this paragraph at all. I just reflow it.)

How about this?

```
Appending a ``'b'`` to the mode opens the file in :dfn:`binary mode`.
Binary mode data is read and written as :class:`bytes` objects.
You must not specify *encoding* when opening file in binary mode.
```